draft-ietf-httpbis-connect-tcp.md

```diff
@@ -187,21 +183,20 @@ If the client infers that classic HTTP CONNECT is not supported, it SHOULD retry
 
 ~~~
 https://$PROXY_HOST:$PROXY_PORT/.well-known/masque
-                    /tcp/{target_host}/{tcp_port}/
+                 /tcp/{target_host}/{target_port}/
 ~~~
 {: title="Registered default template"}
 
 If this request succeeds, the client SHOULD record a preference for "connect-tcp" to avoid further retry delays.
 
 ## Multi-purpose proxies
 
-The names of the variables in the URI Template uniquely identify the capabilities of the proxy.  Undefined variables are permitted in URI Templates, so a single template can be used for multiple purposes.
+A single URI Template can be used for multiple purposes if it includes all the relevant variables.  Variables that are not relevant to a particular usage are left undefined, which is permitted when expanding a URI Template.
 
-Multipurpose templates can be useful when a single client may benefit from access to multiple complementary services (e.g., TCP and UDP), or when the proxy is used by a variety of clients with different needs.
+Multipurpose templates can be useful when a single client may benefit from access to multiple complementary services (e.g., TCP and UDP), or when the proxy is used by a variety of clients with different needs.  The contents of the URI Template are not necessarily sufficient to determine its purpose, so clients must determine this in some other way, such as by probing or via a separate usage indication.
 
 ~~~
-https://proxy.example/{?target_host,tcp_port,target_port,
-                        target,ipproto,dns}
+https://proxy.example/{?target_host,target_port,target,ipproto,dns}
```

Comment:
@bemasc with the rest of the updates in this PR (which I think are good and reflect the discussion), I'm thinking that it is best to remove this multi-purpose proxies section. I don't think it makes sense to have in this particular document. It doesn't seem specific to connect-tcp, and it explicitly mentions that it isn't sufficient to determine the purpose.

Reply:
OK, I've removed this section.